evaluators/text/ReciprocalRankEvaluator/__init__.py

```diff
@@ -14,10 +14,18 @@ def __init__(self, *args, **kwargs):
     def metric(self):
         return f'ReciprocalRank@{self.eval_at}'
 
-    def evaluate(self, actual: Sequence[str], desired: Sequence[str], *args, **kwargs) -> float:
+    def evaluate(self, actual: Sequence[int], desired: Sequence[int], *args, **kwargs) -> float:
```

Comment:
Signature change to Sequence[Any] I think is more appropiate

Reply:
This executor supports only document IDs which will be int. It doesn't support scores which will be float. Do we still want to change it to Any?

Reply:
DocumentIDs may come in form of str, u can do Sequence[Union[str, int]]